src/benchmarks/ci.py

```diff
@@ -89,12 +95,17 @@ def __init__(self, cfg: Config):
 
         self.name = cfg.benchmark.name
         self.input_size = input_size
-        benchmark = scenario(
+
+        kwargs = dict(
             n_experiences=cfg.benchmark.n_experiences,
             seed=cfg.seed,
             train_transform=train_transform,
             eval_transform=eval_transform,
         )
+        if cfg.benchmark.name == "PermutedMNIST":
```

Comment:
PermutedMNIST here is correct? not CIFAR?

Reply:
Actually it should be != PermutedMNIST, as this is the only benchmark when fixed_class_order argument cannot be passed. In other cases I just set the value of this argument to None, so it could be probably updated some time in future if we for example want to fix this order for MNIST of FMNIST.